vllm/model_executor/layers/fused_moe/cpu_fused_moe.py

```diff
@@ -237,7 +239,7 @@ def __call__(
 
 class CPUFusedMOE:
     def __init__(self, layer: torch.nn.Module) -> None:
-        pass
+        self._prepared = False
```

Comment:
this variable will be redundant once you start doing the de-interleaving in the constructor.